deck/models.py

```diff
@@ -247,3 +247,13 @@ def create_initial_jury(sender, instance, signal, created, **kwargs):
     jury.users.add(instance.author)
     instance.jury = jury
     instance.save()
+
+
+@receiver(post_delete, sender=Proposal)
+def send_proposal_deleted_mail(sender, instance, **kwargs):
+    context = {'event_title': instance.event.title,
+               'proposal_title': instance.title}
+    message = render_to_string('mailing/jury_deleted_proposal.txt', context)
+    subject = _(u'Proposal from %s just got deleted' % instance.event.title)
+    recipients = [jury.email for jury in instance.event.jury.users.all()]
```

Comment:
I don't think it makes so much difference in terms of speed/memory.
It will hit the database anyways. Also the list comprehension seems more elegant. 
So...Should I change? It is your choice more than mine. :smile: